src/player.h

```diff
@@ -384,6 +384,8 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
         int mabuff_cut_bonus() const;
         /** Returns true if the player is immune to throws */
         bool is_throw_immune() const;
+        /** Returns false if the player can't be dragged */
+        bool stability_check(int strength) const override;
```

Comment:
This would be more intuitive if it returned true if the player defended against the attack.
Like `uncanny_dodge` or `is_immune` etc.